ckanext/dcatapit/harvesters/csw_harvester.py

```diff
@@ -103,6 +142,12 @@ class DCATAPITCSWHarvester(CSWHarvester, SingletonPlugin):
         }
     }
 
+    _ckan_locales_mapping = {
```

Comment:
If we need to use this default configuration, there should be possible to overwrite the configuration.
please see what we did in [multilang harvester](https://github.com/geosolutions-it/ckanext-multilang/blob/master/ckanext/multilang/harvesters/multilang.py#L165)
Basically what we currently are doing for other config properties [in this harvester](https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/harvesters/csw_harvester.py#L117)

This piece of code can be introduced in utils.py